libcodechecker/libhandlers/store.py

```diff
@@ -315,6 +316,11 @@ def main(args):
 
     try:
         assemble_zip(args.input, zip_file, client)
+
+        if os.stat(zip_file).st_size > MAX_UPLOAD_SIZE:
+            LOG.error("The result list to upload is too big.")
```

Comment:
Write out the maximum uploadable file size too with an exclamation mark at the end of the sentence.